cluster/gce/gci/configure-helper.sh

```diff
@@ -427,6 +427,12 @@ EOF
 multizone = ${MULTIZONE}
 EOF
   fi
+  if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
+    use_cloud_config="true"
+    cat <<EOF >>/etc/gce.conf
+alpha-features = ${GCE_ALPHA_FEATURES}
```

Comment:
ah -- I see now from github output, it looks like it's not aligned due to the extra "+"